elro/api.py

```diff
@@ -240,6 +240,8 @@ async def async_process_command(
                         else None,
                     )
                     self._protocol.datagram_data = self._loop.create_future()
+                    if (raw_data[0].decode("utf-8").strip().casefold() == "{ST_answer_OK}".casefold()):
+                        continue
```

Comment:
```suggestion
                    if (
                        raw_data[0].decode("utf-8").strip().casefold()
                        == "{ST_answer_OK}".casefold()
                    ):
                        continue
```

When does this case happen?